packages/x-date-pickers/src/MobileDateTimePicker/MobileDateTimePicker.tsx

```diff
@@ -101,7 +101,13 @@ export const MobileDateTimePicker = React.forwardRef(function MobileDateTimePick
         autoFocus
         DateInputProps={DateInputProps}
         components={components}
-        componentsProps={componentsProps}
+        componentsProps={{
+          ...componentsProps,
+          tabs: {
+            underlineLocation: 'bottom',
```

Comment:
Why do we need to define it here ?
Couldn't it be the layout that says "the tabs is below so the underline is above" ?
Since it already has the logic

Or if we are changing the rendering order (which appears to be the case in `PickersLayout`, maybe simply use `:last-child` inside the toolbar and remove the prop altogether.

Reply:
My 2nd proposal was keeping the style inside the toolbar itself, but deduce it from the DOM position, not from a prop

Reply:
Hm. I've played around with your second suggestion. Thanks for an option I definitely did not think about.
Indeed, it does seem like the most correct solution, especially when now we are going to have correct tabulation/DOM order. 👍